tests/preserve_sequence_ex1.ron

```diff
@@ -7,14 +7,14 @@
         3: '9',
         4: '1',
         5: '2',
-        6: '3',
+        6: '3'
```

Comment:
This is not desired. Can we change the implementation to only omit the trailing comma for single-line things?

Reply:
I'm not sure what happened here, but I've already made this fix in the last batch of changes